src/arviz_plots/visuals/__init__.py

```diff
@@ -71,6 +71,15 @@ def scatter_x(da, target, backend, y=None, **kwargs):
     return plot_backend.scatter(da, y, target, **kwargs)
 
 
+def scatter_xy(da, target, backend, **kwargs):
+    """Plot a scatter plot x vs y.
+
+    The input argument `da` is split into x and y using the dimension ``plot_axis``.
+    """
+    plot_backend = import_module(f"arviz_plots.backend.{backend}")
```

Comment:
not yet, but after merging the plot_ridge PR, this should use `_process_da_x_y` too. See https://github.com/arviz-devs/arviz-plots/pull/57#discussion_r1664857354. The `x` as aesthetic I mentioned above won't work otherwise.